packages/core/src/browser/keybinding.ts

```diff
@@ -58,7 +58,7 @@ export namespace Keybinding {
 
     /* Determine whether object is a KeyBinding */
     export function is(arg: Keybinding | any): arg is Keybinding {
-        return !!arg && 'command' in arg && 'keybinding' in arg;
+        return !!arg && arg === Object(arg) && 'command' in arg && 'keybinding' in arg;
```

Comment:
JS has [typeof operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) for runtime type checking, i.e. `typeof arg === 'object'` to check whether `arg` is not a primitive or undefined

Reply:
You can also compare the constructors. The subtleties between the different checks are slight AFAICT, but I chose this method to avoid the untyped string 'string'.
I'm happy to change to that method if it's deemed better in some way.

Reply:
The `typeof` check doesn't guard against `null` values, but that is mitigated by the `!!arg` check